framework/src/modules/chain/chain.js

```diff
@@ -402,6 +405,23 @@ module.exports = class Chain {
 			},
 			syncInterval
 		);
+
+		/**
+		 * Periodically calculate consensus
+		 */
+		this.logger.trace('Peers ready');
```

Comment:
Let's make this separate function `_calculateConsensus`.
Also this log is not relevant anymore